index.js

```diff
@@ -8,5 +8,13 @@ module.exports = {
     return {
       fastboot: require('./lib/commands/fastboot')
     };
+  },
+
+  contentFor: function(type, config) {
+    // When using a dist built html file, we need to reinsert 
+    // content-for body so fastboot can insert its content.
+    if (type === 'body') {
+      return "{{content-for 'body'}}";
```

Comment:
@rwjblue good idea.  Also, it will have to be an html comment otherwise that string is gonna show up on the page in a standard ember serve (non-fastboot)